src/net/sourceforge/kolmafia/preferences/Preferences.java

```diff
@@ -1133,6 +1136,16 @@ public static void resetPerAscension() {
 
     // Some need special treatment
     MonorailManager.resetMuffinOrder();
+
+    // Increment the amount of times we've ascended today
+    Preferences.increment("_ascensionsToday");
```

Comment:
This will work here but it's probably put it somewhere more meaningful like in the function that initially tracks an ascension event. Do we want to track it on entering or on leaving Valhalla? What happens if you sit in Valhalla over rollover?